deploy/init-hacbs-user-workspace-sa-rbac.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
based on the latest placement service cabal discussion today with @sbose78 and team, this compute cluster side rbac/sa initialization will most likely be some new api/contract, most likely exposed managed by the pipeline service, but that is not yet "official" as the design discussion that not reach the conclusion of Jira's getting opened, consensus being reached, etc.